packages/accordion/sp-accordion-item.ts

```diff
@@ -10,8 +10,9 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 import { AccordionItem } from './src/AccordionItem.js';
+import { defineElement } from '@spectrum-web-components/shared/src/define-element.js';
```

Comment:
Mochanical point that I could take or leave, not every package relies on `@spectrum-web-components/shared` but every package does rely on `@spectrum-web-components/base`, probably doesn't matter much (especially with the full path to code here), but wanted to make sure we agreed outloud this was a good place for this code to live.